pkg/controller/installation/core_controller.go

```diff
@@ -1499,6 +1516,22 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 	// Tell the status manager that we're ready to monitor the resources we've told it about and receive statuses.
 	r.status.ReadyToMonitor()
 
+	// BPF Upgrade without disruption:
+	// First get the calico-node daemonset.
+	calicoNodeDaemonset = appsv1.DaemonSet{}
+	err = r.client.Get(ctx, types.NamespacedName{Namespace: common.CalicoNamespace, Name: common.NodeDaemonSetName}, &calicoNodeDaemonset)
+	if err != nil {
+		reqLogger.Error(err, "An error occurred when querying the calico-node daemonset")
```

Comment:
Thanks @caseydavenport - as per comment I have replaced the code to be more consistent and more user-friendly as advised but could I maybe tweak the new code ever so slightly so that the new error messages are not exactly the same as existing error messages in order to differentiate them?  WDYT?  Thx